styles/keybinding-resolver.less

```diff
@@ -2,14 +2,24 @@
 @import "octicon-mixins";
 
 .key-binding-resolver {
-  -webkit-flex: 0.25;
   overflow: auto;
 
-  .keystroke {
-    margin-right: 1em;
+  .panel-heading {
+    position: sticky;
```

Comment:
Ohh.. wow. Didn't know that works now. We could use that in other places. Like root folders in the tree-view or package titles in the settings.

Reply:
:heart_eyes: